### PR TITLE
qm-clone: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
#8537

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/qm-clone.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Copy / Clone a virtual machine:
+
+`qm copy {{100}} {{102}}`
+
+- Set a name for the cloned VM:
+
+`qm copy {{100}} {{102}} --name {{name}}`
+
+- Set a description for the cloned VM:
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:14]:
```suggestion
- Copy a virtual machine using a specific descriptionn:
```

File: pages/linux/qm-clone.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`qm copy {{100}} {{102}} --description {{description}}`
+
+- Create a full clone of the source VM:
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:18]:
```suggestion
- Copy a virtual machine creating a full copy of all disks:
```

File: pages/linux/qm-clone.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`qm copy {{100}} {{102}} --full`
+
+- Specify file format for the cloned VM (requires full clone):
+
+`qm copy {{100}} {{102}} --full --format {{qcow2 | raw | vmdk}}`
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:24]:
```suggestion
`qm copy {{100}} {{102}} --full --format {{qcow2|raw|vmdk}}`
```

File: pages/linux/qm-clone.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Set a description for the cloned VM:
+
+`qm copy {{100}} {{102}} --description {{description}}`
+
+- Create a full clone of the source VM:
+
+`qm copy {{100}} {{102}} --full`
+
+- Specify file format for the cloned VM (requires full clone):
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:22]:
```suggestion
- Copy a virtual machine using a specific format for file storage (requires a --full clone):
```

File: pages/linux/qm-clone.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`qm copy {{100}} {{102}} --full --format {{qcow2 | raw | vmdk}}`
+
+- Specify pool for the cloned VM:
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:26]:
```suggestion
- Copy a virtual machine then add it to a specific pool:
```

File: pages/linux/qm-clone.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Copy a virtual machine using a specific descriptionn:
+
+`qm copy {{vm_id}} {{new_vm_id}} --description {{description}}`
+
+- Copy a virtual machine creating a full copy of all disks:
+
+`qm copy {{vm_id}} {{new_vm_id}} --full`
+
+- Copy a virtual machine using a specific format for file storage (requires a --full clone):
```
Comment (navarroaxel) [inline pages/linux/qm-clone.md:22]:
```suggestion
- Copy a virtual machine using a specific format for file storage (requires `--full`):
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**